integration/copy_test.go

```diff
@@ -11,10 +11,13 @@ import (
 )
 
 func init() {
-	check.Suite(&CopySuite{})
+	check.Suite(&CopySuite{
+		ss: &SkopeoSuite{},
```

Comment:
This doesn’t seem awfully shorter than setting up the one registry we need, and doing the copy (and it sets up 4! of them).

Perhaps add a `setupRegistryV2WithPreparedImagesAt` helper, and use it in both suites?